clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -69,6 +70,11 @@
     private final boolean autoCommitEnabled;
     private final AutoCommitTask autoCommitTask;
     private final ConsumerInterceptors interceptors;
+    private final boolean excludeInternalTopics;
+    
+    static {
+        INTERNAL_TOPICS.add("__consumer_offsets");
```

Comment:
@granthenke Would you please make the JIRA ticket for this PR dependent on the proper sub-task of KIP-4 (that would implement the boolean field you mentioned)? That way I can update this PR as soon as the KIP-4 sub-task is finished. Thanks.